src/templates/template-common/utils.py

```diff
@@ -156,7 +171,21 @@ def setup_output_dir(config: Any, rank: int) -> Path:
         path.mkdir(parents=True, exist_ok=True)
         config.output_dir = path.as_posix()
 
-    return Path(idist.broadcast(config.output_dir, src=0))
+    return Path(idist.broadcast(str(config.output_dir), src=0))
+
+
+def setup_config_saving(config):
```

Comment:
Do we need this method again ?

Reply:
No it seems to be left during the merge. Lets remove it